STX.REST.RESTFulSense.Clients/Models/Services/HttpExchanges/Exceptions/InvalidArgumentHttpExchangeException.cs

```diff
@@ -0,0 +1,20 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System;
+using Xeptions;
+
+namespace STX.REST.RESTFulSense.Clients.Models.Services.HttpExchanges.Exceptions
+{
+    public class InvalidArgumentHttpExchangeException : Xeption
```

Comment:
Delete this exception

Reply:
@cjdutoit @glhays This exception is used to wrap the native `ArgumentException` when thrown from the external library. This is part of the exception test cases
https://github.com/The-Standard-Organization/STX.REST.RESTFulSense.Clients/blob/a506e998aff1a27bc9d279428e7e0be05ca70e7b/STX.REST.RESTFulSense.Clients/Services/Foundations/HttpExchanges/HttpExchangeService.Exceptions.cs#L85

Reply:
Ok, you can keep this exception to localise the external exception.